.golangci.yml

```diff
@@ -20,6 +20,7 @@ linters:
    - goconst
    - gochecknoglobals
    - gochecknoinits
+   - gocyclo
```

Comment:
What is the reason behind disabling the `gocyclo` linter?

Reply:
"cyclop" and "gocyclo" are similar linters except that "cyclop" also scans packages (gocycle scnas only functions).

From their github readme:
Gocyclo calculates [cyclomatic complexities](https://en.wikipedia.org/wiki/Cyclomatic_complexity) of functions in Go source code. "Source: https://github.com/fzipp/gocyclo"
Cyclop calculates cyclomatic complexities of functions or packages in Go source code. "Source: https://github.com/bkielbasa/cyclop"

So we decided to disable gocyclo.